apps/rush-lib/src/rush-schema.json

```diff
@@ -112,6 +112,30 @@
       "description": "Hooks are customized script actions that Rush executes when specific events occur.",
       "type": "object",
       "properties": {
+        "preRushInstall": {
+          "description": "The list of scripts to run before the Rush install command starts.",
+          "type": "array",
+          "optional": true,
+          "items": {
+            "type": "string"
+          }
+        },
+        "postRushInstall": {
+          "description": "The list of scripts to run after the Rush install command finishes.",
+          "type": "array",
+          "optional": true,
+          "items": {
+            "type": "string"
+          }
+        },
+        "preRushBuild": {
+          "description": "The list of scripts to run before the Rush build command starts.",
+          "type": "array",
+          "optional": true,
+          "items": {
+            "type": "string"
+          }
+        },
         "postRushBuild": {
```

Comment:
Those events have been checked in for code. But I forgot to update rush-schema.json.

---
In reply to: [119751799](https://github.com/Microsoft/web-build-tools/pull/218#discussion_r119751799) [](ancestors = 119751799)